### PR TITLE
the beginning

### PR DESCRIPTION
**Pull request by Aliaksei-Halkin**

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.exadel.team2.sandbox.entity;
+
+import javax.persistence.Table;
+import javax.persistence.Entity;
+
+@Entity
+@Table(name = "INTERVIEW_FEEDBACK")
+public class InterviewFeedbackEntity {
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java:8]: Need to finish this entity, ``` feedback ```, ``` createdAt ```, ``` updatedAt ``` fields could be added here

**Review by akarzhaneuski — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java (hunk 1 above)
Comment (Aliaksei-Halkin) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java:20]: added

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
+    @GenericGenerator(name = "native", strategy = "native")
+    @Column(name = "IFB_ID")
+    private int id;
```
Comment (ikantsevich) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java:26]: Better use Long instead of int for id

Comment (Aliaksei-Halkin) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/InterviewFeedbackEntity.java]: replaced all "int id" on "long id"

**Comment by ikantsevich** [comment]
Please add controllers and services

**Comment by Aliaksei-Halkin** [comment]
I added controllers and services, changed entity, but I'm not sure in tables' relationships. And app finished with exit cose 1

**Comment by Aliaksei-Halkin** [comment]
opened new PR

**Pull request closed without merging**